apps/ui/components/single-search/SearchForm.tsx

```diff
@@ -105,71 +114,126 @@ const StyledSelect = styled(Select<OptionType>)`
   }
 `;
 
-const Group = styled.div<{ children: ReactNode[]; $gap?: string }>`
-  > div:first-of-type {
-    label {
-      width: calc(${({ children }) => children.length} * 100%);
-    }
-  }
-
-  .inputGroupEnd {
-    & > div {
-      border-left-width: 0;
+const StyledCheckBox = styled(Checkbox)`
+  &&&& {
+    @media (min-width: ${breakpoints.m}) {
+      margin-top: -70px;
+      grid-column: 3 / span 1;
+      grid-row: 4;
     }
-    margin-left: 0;
   }
+`;
 
+const TwInput = styled.div`
+  display: grid;
+  grid-template-columns: 50% 50%;
   .inputGroupStart {
     & > div {
       border-right-width: 0;
     }
+  }
+`;
 
-    & + .inputGroupEnd > div {
-      border-left-width: 2px;
+const OptionalFilters = styled(ShowAllContainer)<{
+  children: ReactNode[];
```

Comment:
Is the children used still?

Reply:
Nope, same as with the other `children` - a relic from a styled-component with a dynamic grid. But I'd rather specify the styles just written out and specify all needed styles myself. We know how many children the `Filters` & `OptionalFilters` have - so why be unnecessarily clever. :P
This is removed now.